src/web/routes/application/address/postcode/postcode.js

```diff
@@ -30,6 +34,10 @@ const auditPostcodeLookup = (config, req, outcome, numberOfResults) => {
 }
 
 const behaviourForPost = (config) => async (req, res, next) => {
+  if (!validationResult(req).isEmpty()) {
+    return next()
+  }
+
```

Comment:
It would be good to split this file up in a subsequent PR - it's slowly growing!!

Reply:
You'll get test failures without it.

Reply:
Yes sure, I didn't mean this part .. it was a general observation on the file. I think the OS places logging could be extracted.